### DISCUSSION TITLE
TreeTable widget?

### DISCUSSION THREAD

**Discussion by AlexKotik** [zauberzeug/nicegui#3146] [Ideas / Feature Requests]
### Question

Sorry, if I'm missing something, but is there a widget that combines tree and table? It seems that there are separate QTree and QTable widgets in Quasar, but neither of them supports recursive table like elements. AGGrid seems to support it only in paid version, free version doesn't, is that so? Here's an example of a widget I'm looking for:
![treetable](https://github.com/zauberzeug/nicegui/assets/24991384/210d1369-27a1-4c58-921c-030457d89969)

**Comment by falkoschindler** [comment]
Currently there is, unfortunately, no tree-table element in NiceGUI.
Let's turn this discussion into a feature request. Maybe someone has an idea how to implement such an element, maybe using existing JavaScript frameworks.

**Reply by dorel14** [reply]
Hello, perhaps by integration of jstree and jstree-table plug-in 
[https://www.jstree.com/](https://www.jstree.com/)

[https://github.com/adamjimenez/jstree-table](https://github.com/adamjimenez/jstree-table)
